common/changes/@itwin/ecschema-metadata/abhi-ecclass-getProperties-issue_2025-04-11-12-07.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/ecschema-metadata",
+      "comment": "**BREAKING:** The signature of protected method `ECClass.buildPropertyCache()` has been changed to take less arguments and return a `Promise<Property[]>`.",
+      "type": "none"
+    }
+  ],
+  "packageName": "@itwin/ecschema-metadata"
+}
```

Comment:
Since this is a protected method that we marked as internal api (in the other PR), I don't think we should add a message for that to the changelog. I don't think we want to allow callers outside this package to re-build the property cache.